libs/dh/market-participant/feature-gridarea/src/lib/details-edit/dh-market-participant-gridarea-edit.component.html

```diff
@@ -0,0 +1,58 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<ng-container
+  *transloco="let t; read: 'marketParticipant.gridAreas.changeGridArea'"
+>
+  <div class="top-btns">
+    <watt-button (click)="openEditModal()">{{ t("editGridArea") }}</watt-button>
+  </div>
+  <ng-container>
+    <watt-modal
+      #nameChangeModal
+      [title]="'Rediger netområde'"
+      [size]="'small'"
+      [disableClose]="gridChangesLoading"
+      closeLabel="Close modal"
```

Comment:
suggestion: Let's use the translation:

```suggestion
      [closeLabel]="t('editGridAreaModalCloseLabel')"
```